src/libcrun/linux.c

```diff
@@ -2214,6 +2214,9 @@ inherit_env (pid_t pid_to_join, libcrun_error_t *err)
   for (str = content; str < content + len; str += strlen (str) + 1)
     if (putenv (str) < 0)
         return crun_make_error (err, errno, "putenv '%s'", str);
+
+  free(content);
```

Comment:
was it reported by static analysis?  This is a false positive, as we need to leak here, the content is used to store the environment variables.

Reply:
Yep, ran fresh covscan against the code. Thanks for the context.